libvast/vast/detail/assert.hpp

```diff
@@ -22,6 +22,7 @@
 #  define VAST_ASSERT(expr)                                                    \
     do {                                                                       \
       if (static_cast<bool>(expr) == false) {                                  \
+        /* NOLINTNEXTLINE */                                                   \
```

Comment:
Did you try using `fmt` instead?
```
fmt::print(stderr, "{}:{}: assertion failed '{}'\n", __FILE__,          \
           __LINE__, #expr);
```

Reply:
I think this is a bad idea. An assertion macro should call as low level of an API as possible.